crates/holochain/Cargo.toml

```diff
@@ -94,7 +90,6 @@ tracing-subscriber = "0.3.16"
 url = "2.4"
 url2 = "0.0.6"
 uuid = { version = "1.8", features = ["serde", "v4"] }
-holochain_wasm_test_utils = { version = "^0.4.0-dev.24", path = "../test_utils/wasm" }
```

Comment:
This makes the biggest difference it seems, something that this does during its build was preventing me building Holochain for Windows with mingw